app/models/patient.rb

```diff
@@ -175,6 +175,44 @@ def destroy_associated_events
     Event.where(patient_id: id.to_s).destroy_all
   end
 
+  def determine_has_other_contact
+    other_contact.present? || other_phone.present? || other_contact_relationship.present?
+  end
+
+  def age_range
+    case age
+    when nil, ''
+      :not_specified
+    when 1..17
+      :under_18
+    when 18..24
+      :age18_24
+    when 25..34
+      :age25_34
+    when 35..44
+      :age35_44
+    when 45..54
+      :age45_54
+    when 55..100
+      :age55plus
+    else
+      :bad_value
+    end
+  end
+
+  def notes_count
+    notes.count
+  end
+
+  def has_special_circumstances
+    has_circumstance = 0
+    special_circumstances.each do |cir|
+      has_circumstance = 1 if cir
+      break
+    end
+    has_circumstance
```

Comment:
right on! Can we cast this as a boolean? We might also want to check that `cir.present?` since I think there are some Vintage Special Circumstances that might be empty strings?

Reply:
is there a nicer way to cast to boolean in rails other than `!!`?